projects/plugins/jetpack/readme.txt

```diff
@@ -242,27 +242,71 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
+- Premium Content block: use a drop down menu rather than two buttons to switch between the guest/subscriber views.
+- Slideshow shortcode: adds the Jetpack namespace to slideshow shortcode CSS class names.
+- Subscribe Block: add pre/post-publish notices.
+- Subscribe Block: block setting updates including name change, display of current subscribers, and a new style option.
+- Tiled Gallery: add background color block setting.
+- Various Blocks: update Contact Info, Markdown, and Tiled Gallery blocks to include margin design tools.
+- VideoPress: updated design of resumable uploader block.
 
 #### Improved compatibility
+- Added support for the `muted`, `controls` and `playsinline` properties on the wpvideo and videopress shortcodes.
+- CLI Tools: ensure WP CLI is present before extending the class.
+- General: clean up use of deprecated FILTER_SANITIZE_STRING constant. Also mark WPCom_Markdown::get_post_screen_post_type as deprecated due to lack of use.
+- Premium Content Block: prevent block from being nested inside itself.
 - Sharing: avoid warnings when the feature is not active on a site that uses the AMP plugin.
+- Various Blocks: remove deprecated attributes from Button components.
 
 #### Bug fixes
+- Assistant: make call-to-actions point to checkout page.
 - Calendly Block: ensure it can be displayed inline when using a block enabled theme.
-- VideoPress: fix broken styles on resumable uploader component.
+- Google Analytics: fix showing an upgrade button with the latest Jetpack security plans.
+- Jetpack: fix missing "Connect User" button after restoring a connection.
+- Jetpack: remove the duplicated `jetpack_relatedposts_returned_results filter`.
+- Payment Blocks: fixes an issue with the upgrade banner being obscured from all payment blocks.
+- Payments: swap JETPACK_VERSION for the correct JETPACK__VERSION.
 - Pay with Paypal Block: fix inconsistent currency formatting.
-- WordAds: fix defaulting to "house ad" mode for new activations.
+- Pay with PayPal widget: enable widget to work in block-based widget editor and full site editor.
+- Premium Content Block: login button now redirects to the correct post instead of to a 404 page when the URL contains characters that have been previously encoded.
+- Premium Content block: when a visitor subscribes, they now see the premium content without needing to reload the page.
 - Publicize: prevent newlines from being stripped from a custom Publicize message in the classic editor.
 - QR Post: improve the inner logo sizing.
+- SEO Tools: ensure Twitter cards get correct description when a site has a blank tagline.
 - Shortlinks: display the shortlinks interface in the block editor for all post types that support shortlinks.
 - Top Posts Widget: display a fallback list of posts to admins when there are no popular posts to display.
+- VideoPress: fix broken styles on resumable uploader component.
+- WordAds: fix defaulting to "house ad" mode for new activations.
+
+#### Bug Fixes
```

Comment:
Looks like #23314 added a heading for "Bug Fixes" rather than "Bug fixes". Changelogger obligingly preserved the two slightly different headings when squashing.